src/main/java/me/shivzee/util/JMailBuilder.java

```diff
@@ -87,6 +87,8 @@ public static JMailTM createAndLogin(String email , String password) throws Logi
 
             }else if(response.getResponseCode() == 422){
                 throw new LoginException("Account Already Exists! Error 422");
+            }else if (response.getResponseCode() == 429){
+                throw new LoginException("Too Many account create request");

```

Comment:
to be specific its `Too many requests! Error 429 Ratelimited`

Reply:
Added